kpm/spec/kpm/unit/account_spec.rb

```diff
@@ -0,0 +1,347 @@
+require 'spec_helper'
+
+describe KPM::Account do
+
+  shared_context 'account' do
+    let(:logger) {logger       = ::Logger.new(STDOUT)
+    logger.level = Logger::FATAL
+    logger}
+    let(:dummy_data_file) {Dir.mktmpdir('dummy') + File::SEPARATOR + 'kbdump'}
+    let(:url) {'http://127.0.0.1:8080'}
+    let(:killbill_api_key) {'bob'}
+    let(:killbill_api_secrets) {'lazar'}
+    let(:killbill_user) {'admin'}
+    let(:killbill_password) {'password'}
+    let(:db_name) {'killbill'}
+    let(:db_username) {'root'}
+    let(:db_password) {'welcome0'}
+    let(:account_class) { described_class.new(nil,[killbill_api_key,killbill_api_secrets],
+                                             [killbill_user, killbill_password],url,
+                                             db_name, [db_username, db_password],logger)}
+     let(:account_id) {'572cc280-1927-49d4-8fb0-d8319d13bfd5'}
+    let(:dummy_account_id) {SecureRandom.uuid}
+    let(:account_id_invalid) {SecureRandom.uuid}
+    let(:dummy_data) {
+      "-- accounts record_id|id|external_key|email|name|first_name_length|currency|billing_cycle_day_local|parent_account_id|is_payment_delegated_to_parent|payment_method_id|time_zone|locale|address1|address2|company_name|city|state_or_province|country|postal_code|phone|notes|migrated|is_notified_for_invoices|created_date|created_by|updated_date|updated_by|tenant_record_id\n"\
+      "999999|#{dummy_account_id}|#{dummy_account_id}|willharnet@example.com|Will Harnet||USD|0||||UTC||||||||||||false|2017-04-03T15:50:14.000+0000|demo|2017-04-05T15:01:39.000+0000|Killbill::Stripe::PaymentPlugin|2\n"\
+      "-- account_history record_id|id|target_record_id|external_key|email|name|first_name_length|currency|billing_cycle_day_local|parent_account_id|payment_method_id|is_payment_delegated_to_parent|time_zone|locale|address1|address2|company_name|city|state_or_province|country|postal_code|phone|notes|migrated|is_notified_for_invoices|change_type|created_by|created_date|updated_by|updated_date|tenant_record_id\n"\
+      "999998|#{SecureRandom.uuid}|999999|#{dummy_account_id}|willharnet@example.com|Will Harnet||USD|0||||UTC||||||||||||false|INSERT|demo|2017-04-03T15:50:14.000+0000|demo|2017-04-03T15:50:14.000+0000|2\n"
+    }
+    let(:cols_names) {dummy_data.split("\n")[0].split(" ")[2]}
+    let(:cols_data) {dummy_data.split("\n")[1]}
+    let(:table_name) {dummy_data.split("\n")[0].split(" ")[1]}
+    let(:obfuscating_marker) {:email}
+
+    let(:mysql_cli) {"mysql #{db_name} --user=#{db_username} --password=#{db_password} -e"}
```

Comment:
This test requires MySQL to be running, which is not always the case (e.g. on Travis). Could you create a new RSpec group (e.g. "test:mysql:spec") in the Rakefile so they are not run by default?